src/index.html

```diff
@@ -1,13 +1,329 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Bose</title>
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:slnt,wght@-10,800;-5,800;0,400;0,700;0,800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body" id="page">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="navigation">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="BOSE logo"
+              >
+            </a>
+
+            <a href="tel:+1 234 555-55-55" class="icon icon--call header__call"></a>
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+          <h1 class="header__title">
+            The world
+            <br>
+            shades.
+            <br>
+            Your music
+            <br>
+            shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav  class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="navigation">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="BOSE logo"
+              >
+            </a>
+
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="/" class="menu__link">
+                Home
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#recommended" class="menu__link">
+                Recommended
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#product-category" class="menu__link">
+                Product category
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#advanteges" class="menu__link">
+                category
+              </a>
```

Comment:
It leads to advantages, so let's change this text to advantages

Reply:
I mixed up the code and later did not replace everything with the right names🤷‍♂️